docs/website/docs/plus/ecosystem/snowflake_plus.md

```diff
@@ -102,21 +103,60 @@ def my_iceberg_table():
   </TabItem>
 </Tabs>
 
-
 ## Configuration
 
 The `snowflake_plus` destination extends the standard Snowflake configuration with additional options:
 
 | Option | Description | Required | Default |
 |--------|-------------|----------|---------|
-| `force_iceberg` | Whether to force the creation of Iceberg tables | No | `False` |
+| `iceberg_mode` | Controls which tables are created as Iceberg tables. Possible values: <ul><li>`"all"`: All tables including dlt system tables are created as Iceberg tables</li><li>`"data_tables"`: Only data tables (non-dlt system tables) are created as Iceberg tables</li><li>`"none"`: No tables are created as Iceberg tables</li></ul> | No | `"none"` |
 | `external_volume` | The external volume to store Iceberg metadata | Yes | None |
 | `catalog` | The catalog to use for Iceberg tables | No | `"SNOWFLAKE"` |
-| `base_location` | Custom base location for Iceberg data in the external volume | No | `<dataset_name>/<table_name>` |
+| `base_location` | Template string for the base location where Iceberg data is stored in the external volume. Supports placeholders like `{dataset_name}` and `{table_name}`. | No | `"{dataset_name}/{table_name}"` |
```

Comment:
Idk why this quotation mark is on a separate line, probably because of table width? 
<img width="934" alt="image" src="https://github.com/user-attachments/assets/98a9dafa-cd59-4978-847b-71ed1394829e" />

This also doesn't look the best in a table, not sure if we can do much about that though 🤔 
<img width="926" alt="image" src="https://github.com/user-attachments/assets/c7cb5d78-bc94-49cf-8115-12ae34629f2c" />

Reply:
good point, so far I haven't found an easy way to fix the wrapping issue with the quote. Let me try to replace table with something simpler.

Reply:
@akelad I've changed the table layout to a plain list of parameters in 68f19ec
Please take a look.